common/history/states/african_states.txt

```diff
@@ -492,7 +476,7 @@ STATES = {
 			}
 		}
 
-		add_claim = c:ETH add_homeland = ethiopian
+		add_claim = c:ETH add_homeland = ethiopian add_homeland = amhara
```

Comment:
amhara is to be consolidated into ethiopian for game performance reasons so there should be no additional homeland.